### DISCUSSION TITLE
Headphone appears as connected, paired and trusted bluetooth audio device, but not in list of audio devices.

### DISCUSSION THREAD

**Discussion by blausand** [blueman-project/blueman#2504] [Q&A]
**blueman:** 2.3.5
**BlueZ:** Befehl nicht gefunden.
**Distribution:**: Linux Mint 23.1
**Desktop environment:** Cinnamon
**Device:** Bowers & Wilkins Px7 S2 headphones

**Problem:** Headphone appears as connected, paired and trusted bluetooth audio device, but not in list of audio devices.
**Expected:** I can use the device without loosing days studying linux internals.
**Note:** Device works fine when connected via USB; both analog AND SPDIF sound devices appear and work fine.

![grafik](https://github.com/user-attachments/assets/b02501e8-3c97-44f6-8135-f85c63069461)

Sorry, but the [Ctrl]-[C] just sucks, you know. It copies single values, not key-value pairs, let alone the whole info-block.
i think the interesting part is that the device is well recognized as audio device but cinnamon-sound doesn't list it.

What can I do?

**Comment by cschramm** [comment]
> **Distribution:**: Linux Mint 23.1

I assume you mean 21.3. That one uses pulseaudio by default. The `pulseaudio.service` or `bluetooth.service` logs might give you some hint on what's wrong. blueman is not involved after initializing the connection. If audio services got connected, pulseaudio (or another audio server) did its Bluetooth thing, so should provide audio devices for the connection.
